ui/app/templates/components/transit-key-action/hmac.hbs

```diff
@@ -6,16 +6,26 @@
     </div>
     <KeyVersionSelect @key={{key}} @onVersionChange={{action (mut key_version)}} @key_version={{key_version}} />
     <div class="field">
-      <label for="input" class="is-label">
-        Input
-      </label>
+      <Toolbar>
+        <label for="input" class="is-label">Input</label>
+        <ToolbarActions>
+          <div class="toolbar-separator"></div>
+          <CopyButton class="button is-transparent" @clipboardText={{input}}
+            @buttonType="button" @success={{action (set-flash-message 'Data copied!')}}>
+              <Icon @glyph="copy-action" aria-label="Copy" />
+          </CopyButton>
+        </ToolbarActions>
+      </Toolbar>
       <div id="input-control" class="control is-relative">
-        <IvyCodemirror @valueUpdated={{action (mut input)}} @options={{hash
-            lineNumbers=true
-            tabSize=2
-            mode='ruby'
-            theme='hashi'
-          }} @data-test-transit-input="input" />
+        <JsonEditor
+            @valueUpdated={{action (mut input)}} 
+            @options={{hash
+              lineNumbers=true
```

Comment:
On these instances where we're swapping out IvyCodemirror for the JsonEditor component, we can simplify the options passed since many of these are in the default settings for the JsonEditor component

Reply:
Good point will fix it.